lib/include/ultrahdr/jpegr.h

```diff
@@ -332,6 +331,12 @@ class JpegR {
 
    bool isUsingMultiChannelGainMap() { return this->mUseMultiChannelGainMap; }
 
+   void setGainMapGamma(float gamma) {
```

Comment:
are these getter setter functions required, as these are only controlled from constructor

Reply:
I think yes. User may change the configurations on the fly since we don't have a `m_sailed`-like flag at jpegr.cpp layer.

Besides, it's not convenient to set this parameters only from constructor. E.g. if user only wants to set gamma while use the default values for the other parameters, they'll need to also provide the other parameter values in the constructor.